src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -314,13 +314,7 @@ private HttpResponseMessage ConvertResponse()
                     responseMessage.SetReasonPhraseWithoutValidation(responseType);
                 }
 
-                bool streamingResponseEnabled = false;
-                if (BrowserHttpInterop.SupportsStreamingResponse())
-                {
-                    _request.Options.TryGetValue(EnableStreamingResponse, out streamingResponseEnabled);
-                }
-
-                responseMessage.Content = streamingResponseEnabled
+                responseMessage.Content = (_request.Options.TryGetValue(EnableStreamingResponse, out var streamingResponseEnabled) ? streamingResponseEnabled : true) && BrowserHttpInterop.SupportsStreamingResponse()
```

Comment:
Since all browsers support response streaming, should we remove `BrowserHttpInterop.SupportsStreamingResponse`? 
- https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream#browser_compatibility
- https://developer.mozilla.org/en-US/docs/Web/API/Response#browser_compatibility